cluster/saltbase/salt/kubelet/default

```diff
@@ -88,8 +88,8 @@
 {% endif -%}
 
 {% set non_masquerade_cidr = "" -%}
-{% if grains.non_masquerade_cidr is defined -%}
-  {% set non_masquerade_cidr = "--non-masquerade-cidr=" + grains.non_masquerade_cidr -%}
+{% if pillar['non_masquerade_cidr'] is defined -%}
```

Comment:
Can you make it `pillar.non_masquerade_cidr` though (instead of `pillar['non_masquerade_cidr']`, to minimize the change here...